crates/biome_css_parser/tests/css_test_suite/error/scss/declaration/ambiguous-selector-recovery.scss

```
@@ -0,0 +1,12 @@
+.test {
+  label:hover {
+    color: ;
+    background: if(style(--scheme: dark): red; else: blue;);
+  }
+  width: 1px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)